src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```